src/app/layout.tsx

```diff
@@ -4,8 +4,8 @@ import { ResetCSS, GlobalCSS } from '@/styles';
 import { i18nConfig } from '@/i18n';
 
 export const metadata: Metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
+  title: 'LeCode',
+  description: 'LeCode interprise is especialized in web development, and also is an agency with React, JavaScript, NodeJs, Electra and NextJs developers',
```

Comment:
a descrição ficaria melhor com algo do tipo: "LeCode Enterprise specializes in web development and operates as an agency with expertise in React, JavaScript, Node.js, Electron, and Next.js."

Reply:
fixed